omnigibson/utils/registry_utils.py

```diff
@@ -273,6 +278,16 @@ def objects(self):
         """
         return list(self.get_dict(self.default_key).values())
 
+    @property
+    def object_names(self):
+        """
+        Get the names of the objects in this registry
+
+        Returns:
+            list of str: Names of the instances owned by this registry
+        """
+        return [obj.name for obj in self.objects]
```

Comment:
should this return a set potentially instead so we can check for membership? (I guess we can just do a normal `registry("name")` query tho, so maybe this isn't important)